srcpkgs/msp430-toolchain/template

```diff
@@ -1,81 +1,51 @@
 # Template file for 'msp430-toolchain'
 pkgname=msp430-toolchain
-version=3.05.00.00
-revision=2
+version=9.3.1.2
+revision=1
 archs="x86_64* i686*"
 create_wrksrc=yes
-hostmakedepends="unzip expect flex tar texinfo"
-makedepends="ncurses-devel"
+hostmakedepends="make gcc binutils bzip2 tar curl flex bison texinfo"
 short_desc="Toolchain for the TI MSP430 microprocessor"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only"
 homepage="http://www.ti.com/tool/msp430-gcc-opensource"
-distfiles="http://software-dl.ti.com/msp430/msp430_public_sw/mcu/msp430/MSPGCC/${version//./_}/exports/msp430-gcc-source.tar.bz2
- http://software-dl.ti.com/msp430/msp430_public_sw/mcu/msp430/MSPGCC/${version//./_}/exports/msp430-gcc-support-files.zip"
-checksum="d2110ec64e8a055ff89b7eef8217b3c233c30e370e2cb43e1ad835dad5907d1c
- 1e2a0903e6b02d5e8aac2a7297ec0685e95a980b13b937b5c7f24f4cf6015407"
+distfiles="https://dr-download.ti.com/software-development/ide-configuration-compiler-or-debugger/MD-LlCjWuAbzH/9.3.1.2/msp430-gcc-9.3.1.11-source-full.tar.bz2
+ https://dr-download.ti.com/software-development/ide-configuration-compiler-or-debugger/MD-LlCjWuAbzH/9.3.1.2/msp430-gcc-support-files-1.212.zip"
+checksum="095197fcd4a81d0df1ea03d62c3c3030f6024ca96df65d828c043bddde634fdd
+ 3b1a39f10a344dfefb767e60ac35becef4c065013be86993195b138a5fb0b8d6"
 nocross=yes
 nostrip=yes
 
 _prefix=/usr/msp430
-_target=msp430-none-elf
 
 do_configure() {
-	unset CC LD AR AS RANLIB CPP CFLAGS CPPFLAGS LDFLAGS
-
-	mkdir obj-msp
-	cd obj-msp
-
-	CFLAGS="-fcommon" CXXFLAGS="-std=c++11" \
-		../sources/tools/configure \
-		--prefix=$_prefix \
-		--enable-languages=c \
-		--program-prefix="msp430-" \
-		--disable-nls \
-		--enable-gold \
-		--without-x \
-		--target=$_target
-	rm -rf tcl
+	cd ${wrksrc}/msp430-gcc-9.3.1.11-source-full
+	chmod +x README-build.sh
```

Comment:
this version of the msp430-toolchain contains a build script that, instead of copying and pasting the steps in here, it's just easier to patch it above and simply run it